bitnami/seaweedfs/templates/tls-secret.yaml

```diff
@@ -0,0 +1,180 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if and .Values.security.enabled .Values.security.mTLS.enabled (not .Values.security.mTLS.autoGenerated.useCertManager) -}}
+{{- if .Values.security.mTLS.autoGenerated.useHelm -}}
+{{- $ca := genCA "seaweedfs-ca" 365 }}
+{{- $releaseNamespace := include "common.names.namespace" . }}
+{{- $clusterDomain := .Values.clusterDomain }}
+{{- $cert := genSignedCert "ca" nil nil 365 $ca }}
```

Comment:
This is not needed. You are re-signing the CA certificate

Reply:
done at https://github.com/bitnami/charts/pull/24944/commits/0b4ab8720340b0a0ce7b3d3c97a98bd4fbff7300